app/pages/sections/routes.test.js

```diff
@@ -683,5 +690,36 @@ describe('section routes', () => {
           expect(res.text.includes('data-test-id="error-title"')).toEqual(false);
         });
     });
+
+    it('should return the correct status and text if response.success is true but apiResponse.success is false', async () => {
```

Comment:
Change these to say `when the FE validation  is successful but API call returned an error`